File: 02-programacion-basica/03-operadores-y-conversiones/java/soluciones/RetoDuracion.java
public class RetoDuracion {
    public static void main(String[] args) {
        int minutosTotales = 145;
        int horas = minutosTotales / 60;
        // % conserva los minutos que no completan una hora.
        int minutos = minutosTotales % 60;
        boolean superaDosHoras = minutosTotales > 120;

        System.out.printf("%d h %d min | supera 2 h: %s%n", horas, minutos, superaDosHoras);
    }
}

File: 02-programacion-basica/03-operadores-y-conversiones/scala/soluciones/RetoDuracion.scala
@main def retoDuracion(): Unit =
  val minutosTotales = 145
  val horas = minutosTotales / 60
  // % conserva los minutos que no completan una hora.
  val minutos = minutosTotales % 60
  val superaDosHoras = minutosTotales > 120
  println(s"$horas h $minutos min | supera 2 h: $superaDosHoras")
